beets/autotag/hooks.py

```diff
@@ -444,7 +500,7 @@ def update(self, dist):
 
     # Adding components.
 
-    def _eq(self, value1, value2):
+    def _eq(self, value1: Union['Distance', Patterntype], value2) -> bool:
```

Comment:
I did that because I kind of assumed that the function would be used similarly to the `__eq__` function. It does seem to be Any based on the uses and other typings.